packages/@glimmer-workspace/build/lib/config.js

```diff
@@ -315,7 +318,29 @@ export class Package {
         commonjs(),
         nodeResolve(),
         ...this.replacements(env),
-        ...(env === 'prod' ? [terser()] : []),
+        ...(env === 'prod'
+          ? [
+              terser({
```

Comment:
for debugging, we would need to tweak this before merging -- at _least_ removing mangle: false